backend/capellacollab/sessions/injection.py

```diff
@@ -3,42 +3,60 @@
 
 import logging
 
+import pydantic
 import requests
 
 from capellacollab import core
 from capellacollab.config import config
+from capellacollab.core import pydantic as core_pydantic
 
 log = logging.getLogger(__name__)
 
 
-def get_last_seen(sid: str) -> str:
-    """Return project session last seen activity"""
+# Has to be in here because of circular imports
```

Comment:
In case of circular imports, we usually just create a `models2.py`. This avoid a circular imports and is more consistent with the existing structure.